tasks.py

```diff
@@ -178,29 +178,29 @@ def i18nc():
 
 
 @task
-def assets(progress=False):
+def assets_build(progress=False):
     '''Install and compile assets'''
     header('Building static assets')
-    cmd = 'webpack -c --config {0}.js'
+    cmd = 'npm run assets:build -- --config {0}.js'
     if progress:
         cmd += ' --progress'
     nrun(cmd.format('webpack.config.prod'), pty=True)
```

Comment:
I think `nrun` should be replaced by `lrun`. It's only purpose was to replace to transform from `nrun('my-cmd')` to `lrun('node_modules/.bin/my-cmd')`.